libensemble/gen_classes/gpCAM.py

```diff
@@ -61,7 +64,13 @@ def __init__(self, H, persis_info, gen_specs, libE_info=None):
         self.my_gp = None
         self.noise = 1e-8  # 1e-12
 
-    def ask(self, n_trials):
+    def ask(self, num_points: Optional[int] = 0) -> List[dict]:
```

Comment:
I guess these wrappers could go at a higher level, though may reduce visibility to end users. 
Could be base class -> libE class -> generators.